deploy-agent/deployd/agent.py

```diff
@@ -120,6 +122,22 @@ def serve_build(self):
         if not self._executor:
             self._executor = Executor(callback=PingServer(self), config=self._config)
         # start to ping server to get the latest deploy goal
+        if len(self._envs) > 0:
+            for status in self._envs.values():
+                # for each container, we check the health status
+                try:
+                    cmd = ['docker', 'inspect', status.report.envName]
+                    output = subprocess.run(cmd, check=True, stdout=subprocess.PIPE).stdout
```

Comment:
you can use this cmd `docker inspect -f '{{.State.Health.Status}}'`

Reply:
sure, thanks